.github/workflows/ci.yaml

```diff
@@ -15,20 +19,136 @@ on:
       - cryostat-v[0-9]+.[0-9]+
```

Comment:
We will need to change to the event to `pull_request_target` to ensure workflow can request write permissions.